python/sbp/jit/parse.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2019 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+import glob
+import ntpath
+import os
+import shutil
+import importlib
+
+import numpy as np
+import numba as nb
+
+from numba.pycc import CC
+
+from sbp.constants import SENDER_ID as _SENDER_ID
+from sbp.constants import SBP_PREAMBLE as _SBP_PREAMBLE
+from sbp.constants import _crc16_tab
+
+from distutils.ccompiler import CCompiler
+from numpy.distutils.misc_util import get_num_build_jobs
+from numpy.distutils.ccompiler import _global_lock, _processing_files
+from numpy.distutils import log
+
+import sys
+import types
+
+# monkeypatch numpy.distutils.ccompiler. Parallel build hangs with py27
+def replace_method(klass, method_name, func):
+    if sys.version_info[0] < 3:
+        m = types.MethodType(func, None, klass)
+        setattr(klass, method_name, m)
+
+def CCompiler_compile(self, sources, output_dir=None, macros=None,
+                      include_dirs=None, debug=0, extra_preargs=None,
+                      extra_postargs=None, depends=None):
+    """
+    Compile one or more source files.
```

Comment:
I think we can delete their docs, and maybe just insert a link to the original source?